src/main/java/com/google/sps/utils/CookieUtils.java

```diff
@@ -53,4 +53,13 @@ public static Cookie getCookieFromName(Cookie cookies[], String name){
         }
         return null;
     }
+    public static Cookie createSessionIDCookie(HttpSession oldSession, String newSessionID){
+        if(oldSession != null){
+            oldSession.invalidate();
+        }
+        Cookie cookie = new Cookie("SessionID", newSessionID);
```

Comment:
There's a constant for "SessionID" somewhere, right? Maybe in UserUtils. It'd be good to use that constant here.